dockerfiles/buildall.sh

```diff
@@ -88,7 +88,7 @@ if [[ -z "$BUILD_ML" ]]; then
   build_image platform-build
   build_image platform-core
 else
-  build_image platform-core # NB: if building ML images and core has already been built, this will only pull it
+  docker pull ${DOCKER_REPO}/platform-core:${$BUILD_LABEL}
```

Comment:
was this changed, because was it was already assumed (relied on) platform-build existing anyway?
so this is just concreting the fact that it assumed previous stages (inc. platform-core) were pre-build before building ml images.